### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -9,8 +9,13 @@ source _common.sh
...
 
 #=================================================
-# INSTALL DEPENDENCIES
+# WATCHDOG DEVICE CHECK
 #=================================================
+ynh_script_progression --message="Checking the hardware watchdog device presence..."
+
+if [ ! -f /dev/watchdog ]; then
+    ynh_die --message="Your server doesn't seem to have a '/dev/watchdog' device, this certainly means that it has no hardware watchdog device. However, this package is only relevant if such a device exists, we're sorry."
+fi
```
Comment (Salamandar) [inline scripts/install:17]: This can't pass CI…

Comment (OniriCorpe) [reply scripts/install:17]: oupsi my bad 😬

Comment (Salamandar) [reply scripts/install:17]: no prob :D

**Pull request merged**